shim.js

```diff
@@ -0,0 +1,4 @@
+// see: https://github.com/facebookincubator/create-react-app/issues/3199
```

Comment:
👍  Love when we do that :)

Reply:
It's already fixed in Jest, just not released yet; see https://github.com/facebook/jest/issues/4545#issuecomment-332762365